app/assets/javascripts/member-facing/redirectors.js

```diff
@@ -0,0 +1,43 @@
+const RegisterMemberRedirector = {
+  attemptRedirect:  function(followUpUrl, member) {
+
+    if(typeof(member) !== 'object') {
+      member = window.champaign.personalization.member;
```

Comment:
There's always the possibility of passing the in the params, I was just trying to avoid code duplication in the templates. I'm not feeling strongly about either way, I don't mind changing it if you feel works better.